web/src/components/utils.js

```diff
@@ -0,0 +1,4 @@
+// this is file for sharing variables
+module.exports = {
+    clientID: "325103706675-9g44spipbpiujdkopvs8n3hvh188k14i.apps.googleusercontent.com" // Our Google API Service ID
```

Comment:
Is this the API key? Then this needs to be hidden in a secure place, not checked into the repository. 

https://www.freecodecamp.org/news/how-to-securely-store-api-keys-4ff3ea19ebda/

Now that it's already on the github commit history, we'll have to regenerate this. OK to add a TODO to fix this in a subsequent PR. Another way to do this is via environment variables and have everyone on the team set one up. 

https://gist.github.com/derzorngottes/3b57edc1f996dddcab25